packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -179,8 +195,12 @@ export default class ChildProcessWorker implements WorkerInterface {
     }
   }
 
-  onExit(exitCode: number) {
-    if (exitCode !== 0) {
+  private _onExit(exitCode: number) {
+    if (
+      exitCode !== 0 &&
+      exitCode !== 137 && // SIGKILL
+      exitCode !== 143 // SIGTERM
```

Comment:
be nice to have an exit code constant

Reply:
Inside of `jest-worker` or somewhere shared? Not sure if we'd need it in any other package

Reply:
agree inside of jest workers, i don't think we manually spawn processes much otherwise